editor.planx.uk/src/@planx/components/PlanningConstraints/model.ts

```diff
@@ -5,6 +5,7 @@ export interface PlanningConstraints extends BaseNodeData {
   description: string;
   fn: string;
   disclaimer: string;
+  dataValues?: string[] | undefined;
```

Comment:
Feel quite strongly that `datasets` is NOT a good name because we're not allowing users to say "I only want to check TPO _orders_ dataset from Planning Data, not TPO _zones_" - rather we're letting them select/deselect our pre-curated "options" (each with a title, passport value, and associated source datasets). 

Perhaps `options` is more generic and easier to grasp here though? In my mind, they're basically the PlanningConstraint node's equivalent of Question/Checklist "edges"/"options" ?

Reply:
As for the "all" question - I think this becomes much simpler in a word of Templates (eg PlanningConstraints will _not_ be a "placeholder" node therefore just updated once per service template & changes propagated to children on publish), and if still too cumbersome then would favor data migration script like you say! 

Also no near-future requests for more datasets from Planning Data besides Planning History which I think will have a strong argument for it's own component or API outside of existing `/gis`.